pkgs/applications/misc/visidata/default.nix

```diff
@@ -46,5 +46,6 @@ buildPythonApplication rec {
     maintainers = [ lib.maintainers.raskin ];
     platforms = with lib.platforms; linux ++ darwin;
```

Comment:
```suggestion
```
not required to be set here.

Reply:
I dont understand. Why ?

Reply:
buildPythonApplication sets it to this already.

Reply:
updated